lib/recipe/view/recipe_page.dart

```diff
@@ -2,31 +2,44 @@ import 'package:flutter/material.dart';
 
 import 'package:gibsonify/recipe/recipe.dart';
 
+enum SelectedRecipeScreen {
+  probeScreen,
+  ingredientScreen,
+  detailScreen,
+}
+
 class RecipePage extends StatefulWidget {
```

Comment:
Again we could think whether this should be stateful widget or we move the selectedScreen to the state of the bloc. But this is definitely good for now.

Reply:
Yeah I was thinking about this. For recipes, there's no functional reason to store the state of the selected screen since the page that it goes to should be fixed regardless of where it was last left.

Reply:
Alrighty, that makes sense 👍.